### PR TITLE
Add app config to specify default attachment folder location

### PR DESCRIPTION
**Pull request by tcitworld**
Needs tests & mention in docs

### 🏁 Checklist

- [ ] ⛑️ Tests (unit and/or integration) are included
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

**Review by vitormattos — DISMISSED** [review]
All app config need to be added to file https://github.com/nextcloud/spreed/blob/master/docs/settings.md

**Comment by nickvergessen** [comment]
@tcitworld want to finish this? Or should Vitor take over?

**Comment by tcitworld** [comment]
I'm not sure I'll get some time before end of the vacation, so please take over if you have time to finish this!

File: docs/settings.md [hunk 1]
```diff
@@ -21,10 +21,10 @@
...
-| `attachment_folder`   | `config => attachments => folder` | `/Talk` | Path owned by the user to store uploads and received shares. It is created if it does not exist.         |
-| `read_status_privacy` | `config => chat => read-privacy`  | `0`     | One of the read-status constants from the [constants list](constants.md#participant-read-status-privacy) |
+| Key                   | Capability                        | Default                                          | Valid values                                                                                             |
+|-----------------------|-----------------------------------|--------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| `attachment_folder`   | `config => attachments => folder` | Value of app setting `default_attachment_folder` | Path owned by the user to store uploads and received shares. It is created if it does not exist.         |
```
Comment (nickvergessen) [inline docs/settings.md:26]: Value of app config ...

File: docs/settings.md [hunk 2]
```diff
@@ -94,3 +94,4 @@ Option legend:
...
 | `conversations_files_public_shares`  | string<br>`1` or `0`                                             | `1`        | 🖌️    | Whether the public share integration is enabled allowing to start conversations in the right sidebar on the public share page (Requires `conversations_files` also to be enabled) |
 | `enable_matterbridge`                | string<br>`1` or `0`                                             | `0`        | 🖌️    | Whether the matterbridge integration is enabled and can be configured                                                                                                             |
+| `default_attachment_folder`          | string                                                           | `\Talk`    |        | Specify default attachment folder location                                                                                                                                        |
```
Comment (nickvergessen) [inline docs/settings.md:97]: Please sort by type, so after `bridge_bot_password`

Comment (nickvergessen) [reply docs/settings.md:97]: Also we use unix paths not windows...
```suggestion
| `default_attachment_folder`          | string                                                           | `/Talk`    |        | Specify default attachment folder location                                                                                                                                        |
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by nickvergessen — APPROVED** [review]

**Review by tcitworld — COMMENTED** [review]
Can't approve, but ✅

**Pull request merged**